src/descrambler/constcw.c

```diff
@@ -342,7 +342,7 @@ const idclass_t caclient_ccw_aes_class =
 {
   .ic_super      = &caclient_class,
   .ic_class      = "caclient_ccw_aes",
-  .ic_caption    = N_("AES Constant Code Word Client"),
+  .ic_caption    = N_("AES constant code word client"),
```

Comment:
Like "Conditional Access Client", above, I don't really know how these terms are used. They don't look like proper nouns, so lower case would be correct.